src/hooks/useErrorInput.tsx

```diff
@@ -0,0 +1,15 @@
+import FormError from '../components/FormError/FormError';
+
+const useErrorInput = (error: string) => {
```

Comment:
It's not a hook. It's a component. Also, You can avoid "else if" by splitting logic to separate function that will return error text and then use it in your FormError